service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java

```diff
@@ -41,9 +46,13 @@
 class CaseInitiationControllerTest {
 
     private static final String AUTH_TOKEN = "Bearer token";
-    private static final String USER_ID = "10";
-    private static final String CASE_ID = "1";
+    private static final String SERVICE_AUTH_TOKEN = "Bearer service token";
+    private static final String SYSTEM_USER_USERNAME = "fpl-system-update@mailnesia.com";
+    private static final String SYSTEM_USER_PASSWORD = "Password12";
```

Comment:
use spring values! there is a SystemUpdateUserConfiguration already :)

Reply:
done